src/state/slices/opportunitiesSlice/resolvers/foxfarming.ts

```diff
@@ -70,14 +70,20 @@ export const foxFarmingLpMetadataResolver = async ({
   })
   const apy = bnOrZero(calculatedApy).div(100).toString()
   const reserves = await uniV2LPContract.getReserves()
+
+  // Getting the ratio of the LP token for each asset
+  const totalSupply = (await uniV2LPContract.totalSupply()).toString()
+  const foxReserves = bnOrZero(bnOrZero(reserves[1].toString()).toString())
+  const ethReserves = bnOrZero(bnOrZero(reserves[0].toString()).toString())
```

Comment:
Looks like a bunch of useless `new BN()` and `toString()` conversions but we need it for divisions precision